addons/test_mail/tests/test_performance.py

```diff
@@ -531,7 +531,7 @@ def test_mail_composer_w_template(self):
         test_record, test_template = self._create_test_records()
         test_template.write({'attachment_ids': [(5, 0)]})
 
-        with self.assertQueryCount(admin=28, employee=28):  # tm: 22/22
+        with self.assertQueryCount(admin=29, employee=29):  # tm: 23/23
```

Comment:
Don't forget to add an explanation about increased queries in commit message (always a good practice to be complete when changing those)